src/modules/AnnotationModule.ts

```diff
@@ -43,20 +43,23 @@ export interface AnnotationModuleAPI {
   deleteAnnotation: Highlight;
   selectedAnnotation: Highlight;
 }
-
 export interface AnnotationModuleConfig {
+  initialAnnotationColor: string;
+  api: AnnotationModuleAPI;
```

Comment:
yes, i can explain that:

when loading the reader, you have for each module a section, like the tts section, which has it's own api section, as well as other configuration elements like autoScroll, color etc. 
when using the color in the tts module it would be this.config.initialAnnotationColor, and the api this.config.api. and this would be across all modules, 
on the higher level the api is for the navigator itself. 
        D2Reader.load({
           .....
            tts: {
                enableSplitter: true,
                color: "orange",
                autoScroll: true,
                api: {
                    started: function () {
                    },
                    stopped: function () {
                    },
                    paused: function () {
                    },
                    resumed: function () {
                    },
                    finished: function () {
                    },
                    updateSettings: function (settings) {
                    }
                }
            },
            search: {
                color: "#fce300",
                current: "#ed7138",
                api: {

                }
            },
            bookmarks: {
                api: {
                    addBookmark: function (bookmark) {
                    },
                    deleteBookmark: function (bookmark) {
                    }
                }
            },
            api: {
                getContent: function (href) {
                },
                updateCurrentLocation: function (location) {
                },
                updateSettings: function (settings) {
                }
            },
            injectables: injectables,
            userSettings: userSettings,
            useLocalStorage: true // true = uses local storage, false = uses session storage (default)
        }).then(instance => {

Reply:
but i sure see you r point about renaming config, properties etc.

Reply:
Ok I see what you're saying... there is the `api` of the navigator, and then the `api` of the individual modules, and that they need to be distinct. I think this all makes sense now. The one thing that I think would make it simpler maybe is to just get rid of the `AnnotationsModuleProperties` interface since it's not really needed. The `AnnotationsModule` is the only thing using it, and you already have the properties typed directly on the class itself, so it's not really necessary. Same can be said of the other `XXXModuleProperties` interfaces. Up to you though, it all makes sense to me now as is anyway

Reply:
does this make more sense ? https://github.com/d-i-t-a/R2D2BC/pull/170